src/coord/src/timestamp.rs

```diff
@@ -302,7 +302,14 @@ fn generate_ts_updates_from_debezium(
 ) {
     if let Value::Record(record) = value {
         // All entries in the transaction should have the same timestamp
-        let results = parse_debezium(record);
+        let results = match parse_debezium(record) {
+            Ok(result) => result,
+            Err(e) => {
+                error!("Failed to parse debezium transaction msg: {:?}", e);
+                None
+            }
+        };
+
```

Comment:
Before in various cases we would panic if the format was not what we expected; now we print an error message. What will the behavior of the system be in those cases; do you know? Will we hang and fail to advance the timestamp forever? (If so, that might not be much better than panicking)

Reply:
When we print an error message, we will move on to processing the next message. This PR won't cause the timestamp thread to block or fall behind. The next effect of this PR is that instead of panic'ing on some types of avro decoding failure, we instead print an error message on all types of decode failure.

Reply:
yeah but thinking about it more -- moving on to process the next transaction metadata message is not correct because then sources actually reading the data will either

- incorrectly attribute data at the wrong offsets to the wrong transaction (today). imagine transaction A had 1 message, B had 2 messages, and C had 3 messages -- if you error and skip over B in the current world it won't end well

- even if we skip the transaction and the associated data, we might not line up debezium before / afters correctly (following the previous example, if you skip over the messages in transaction B you might still reference some of those in the `before` fields in other transactions)

I think the appropriate thing to do is to print an error message and then not process any more messages permanently

Reply:
Given the above - I definitely don't think this should merge as is. @cirego what do you think?